packages/evo-blockmodel/pyproject.toml

```
@@ -0,0 +1,74 @@
+[project]
+name = "evo-blockmodel"
```

Per our naming convention I think `blockmodels` should be pluralized here, and in the module name as well

```suggestion
name = "evo-blockmodels"
```